node/src/external/jxcore/DownloadJxcoreLib.ps1

```diff
@@ -0,0 +1,22 @@
+$saveProgressPreference = $ProgressPreference
+$ProgressPreference = "SilentlyContinue"
+
+$jxcoreLibUri = "https://github.com/jasongin/jxcore/releases/download/v0.3.1.2/libjxcore.zip"
```

Comment:
>jasongin/j [](start = 36, length = 10)

What is the right approach here (git location) ? can we check-in the jxcore libs ?